csharp/autobuilder/Semmle.Autobuild.CSharp/DotNetRule.cs

```diff
@@ -211,7 +213,7 @@ private static BuildScript DownloadDotNetVersion(IAutobuilder<AutobuildOptionsSh
 
                         var removeScript = new CommandBuilder(builder.Actions).
```

Comment:
We could make this script run conditionally based on the `out` argument to `GetTemporaryWorkingDirectory`.